webpagetest.js

```diff
@@ -45,17 +45,6 @@ let wptOptions = {
     "pollResults": 25,
     "timeout": 600,
     "timeline": true,
-    customMetrics: [
-        '[memory]',
-        'return new Promise((resolve) => { performance.measureUserAgentSpecificMemory().then((value) => { resolve(value.bytes); }); });',
-        '[pageLinks]',
```

Comment:
Every test returns pageLinks which are later required for crawling, the other metrics can be removed.